.gitignore

```diff
@@ -83,6 +83,7 @@ instance/
 
```

Comment:
Please add autoapi go .dockerignore as well

Reply:
First, this file must be created in a separate PR.